app/assets/stylesheets/color_definitions.scss

```diff
@@ -111,7 +111,7 @@
   --hljs-attribute: #{$hljs-attribute};
   --hljs-symbol: #{$hljs-symbol};
   --hljs-bg: #{$hljs-bg};
-
+  --hljs-builtin-name: #{dark-light-choose(var(--tertiary-high), #65b9db)};
```

Comment:
The `dark-light-choose` function should be moved to color_transformations.scss, for consistency (much like the other `hljs-` variables. Also for consistency, let's use the SCSS variable (`$tertiary-high`) instead of the custom property (in all other files, we should use the custom property).

Reply:
I thought the function _only_ worked in the color_definitions file? Or so I've been told… Ok will change to the regular var.

Reply:
For themes/plugins, yes, those functions only work in `color_definitions.scss` files. In core, they need to be in `color_transformations.scss` because we have some special handling that allows themes to override the defined colors from their own color schemes. (Granted, with custom properties, this is no longer all that relevant.)

Reply:
Aha, thanks for the clarification!